### PR TITLE
Updating examples

### PR DESCRIPTION
**Pull request by src-code**
Updating with new config syntax

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by src-code** [comment]
Note, this example exposes a bug in how `!important` is applied to rules that don't use suffix values: issue #115

**Comment by renatoi** [comment]
LGTM

File: examples/example-config.js [hunk 1]
```diff
@@ -1,424 +1,44 @@
...
-        start: true,
-        end: true,
-        c: true,
-        j: true,
-        m: true
-    },
-
-    // rule
-    'bfc': true
+    'custom': {
```
Comment (redonkulus) [inline examples/example-config.js]: Can we add a comment about what these are?

File: examples/example-config.js [hunk 2]
```diff
@@ -1,424 +1,44 @@
...
+    'classNames': [
+        // normal
+        'Td-u',
+        // pseudos
+        'Td-u:h',
+        // breakpoints
+        'Td-u--sm',
+        // custom
+        'H-uh',
+        'C-primary',
```
Comment (redonkulus) [inline examples/example-config.js]: Add a comment that maps "primary" to the custom variables above.

**Comment by renatoi** [comment]
@redonkulus @src-code I added more descriptive comments about what each thing does. let me know if that's too much or incorrectly written.

**Comment by redonkulus** [comment]
:shipit:

**Pull request merged**